### DISCUSSION TITLE
Bridgetown 1.0 released (with Roda integration)

### DISCUSSION THREAD

**Discussion by jaredcwhite** [jeremyevans/roda#255] [General]
Hey all, I'm excited to share that [Bridgetown 1.0](https://www.bridgetownrb.com), a Ruby-based progressive site generator, is now in wide release, and it ships with Roda integration for dynamic SSR routes (and at the very basic level Roda handles static file serving of Bridgetown build artifacts in its dev server).

More information is available here (this is a cursory intro, I'll have much more to document and share in the months ahead):
https://www.bridgetownrb.com/docs/routes

I'm actually in the middle of porting an RSS news reader web app over from Rails to Bridgetown + Roda, and since it'll be open source I look forward to sharing that repo down the road as an example of this approach.

Let me know if you have any questions or thoughts!

P. S. I enjoyed your RubyConf Pakistan talk on Roda, Jeremy!

P. P. S. The "live reload" functionality in Bridgetown's dev server ended up utilizing Roda's streaming plugin and SSE in the browser—that was a fun project to figure out!

**Comment by jeremyevans** [comment]
Looks good! The only recommendation I have would be to eliminate the `require "bridgetown-routes"` step. It shouldn't be necessary if the `bridgetown_routes` plugin is stored at the expected location (requirable by `roda/plugins/bridgetown_routes`).

When you get the RSS news reader ported, please send a pull request to add it as an example to the documentation page.  It would be great to have more examples for people to examine.

**Reply by jaredcwhite** [reply]
Thanks Jeremy! I'll look into the plugin require tip you suggested.
